### ISSUE TITLE
Extension AWS Toolkit installs however icon does not shows on the side bar

### ISSUE THREAD

**Issue by rivastomas** [aws/aws-toolkit-vscode#3698]
## Problem
Extension AWS Toolkit installs however icon does not shows on the side bar

## System details (run the `AWS: About Toolkit` command)

-   OS: Windows 10 Enterprise
-   Visual Studio Code version: 1.80.2 (use setup)
-   AWS Toolkit version: v1.82.0

** see attached aws-toolkit-vscode logs **

[aws_toolkit_20230731T112256.log](https://github.com/aws/aws-toolkit-vscode/files/12216596/aws_toolkit_20230731T112256.log)

**Comment by rivastomas** [comment]
Additional information after running aws: about toolkit command

![image](https://github.com/aws/aws-toolkit-vscode/assets/11548665/dd46949a-f09b-4235-ac23-0a7f5479c54a)

**Comment by justinmk3** [comment]
Right-click on a blank area of the sidebar. Does it show an "AWS" item and is it checked?

![image](https://github.com/aws/aws-toolkit-vscode/assets/55561878/60aacc19-db81-4192-b7bd-d9abcebf2794)

**Comment by rivastomas** [comment]
No, it does not show the AWS Toolkit icon even though it indicates the AWS Toolkit extension is installed

![image](https://github.com/aws/aws-toolkit-vscode/assets/11548665/ca0815a6-9f2e-4750-8149-58fd32dbeaf8)

![image](https://github.com/aws/aws-toolkit-vscode/assets/11548665/36686bec-22c2-4301-8f19-8021d5971ab3)

**Comment by rivastomas** [comment]
vscode "secondary side bar" loads towards the right side and it is currently empty

![image](https://github.com/aws/aws-toolkit-vscode/assets/11548665/4d3af916-a264-422c-983f-d4ecc8512ef8)

**Comment by justinmk3** [comment]
Can you try `View: Show AWS` vscode command?

I can't find any logs related to this in the main vscode Output channels, nor on their issue tracker. 

If that fails, as a troubleshooting step, try this (which resolved a similar issue https://github.com/aws/aws-toolkit-vscode/issues/3044 ):

1. Close vscode (all instances)
2. Rename vscode's settings and state directories. (This is just a temporary troubleshooting step, you can restore the directories later if needed.)
    - on Windows: `%APPDATA%\Code\User\`
    - [#3044](https://github.com/aws/aws-toolkit-vscode/issues/3044) also mentions `C:\users\<user>\.vscode` , so if that exists, rename it also.
3. Restart vscode
